src/frontend/src/flows/addDevice/manage/verifyTentativeDevice.ts

```diff
@@ -2,7 +2,6 @@ import { html, render } from "lit-html";
 import { AuthenticatedConnection } from "../../../utils/iiConnection";
 import { withLoader } from "../../../components/loader";
 import { renderManage } from "../../manage";
-import { hasOwnProperty } from "../../../utils/utils";
```

Comment:
Is there a reason to not delete the `hasOwnProperty` function?

Reply:
Yeah, I couldn't get rid of it in the `utils` module just yet